app/build.gradle

```diff
@@ -145,6 +145,13 @@ dependencies {
     //Lottie for Splash Screen
     implementation 'com.airbnb.android:lottie-compose:4.2.0'
 
+    // Moshi Converter
+    def moshiVersion = "1.14.0"
+    implementation("com.squareup.moshi:moshi:$moshiVersion")
+    kapt("com.squareup.moshi:moshi-kotlin-codegen:$moshiVersion")
```

Comment:
Is this Moshi setup correct? 

The Dagger module uses the `kotlin.reflect.KotlinJsonAdapterFactory` and normally we would choose one and only one of reflection or code generation.

Reply:
Hi, included the dependency in error, removed